src/linear_algebra/eigenspace.lean

```diff
@@ -242,5 +248,70 @@ begin
       exact h_lμ_eq_0 μ h_cases } }
 end
 
+/-- The generalized eigenspace for a linear map `f`, a scalar `μ`, and an exponent `k ∈ ℕ` is the
+    kernel of `(f x - μ • id) ^ k`. -/
```

Comment:
```suggestion
    kernel of `(f x - μ • x) ^ k`. -/
```
? I didn't open this in Lean to check definitions but is what you wrote type correct?